airbyte-webapp/src/views/layout/SideBar/components/SidebarDropdownMenu.module.scss

```diff
@@ -14,29 +14,23 @@
   width: 100%;
   background: transparent;
   border: 0;
-  outline: none;
   cursor: pointer;
   border-radius: variables.$border-radius-md;
-  color: colors.$grey-30;
+  color: inherit;
```

Comment:
:information_source: Making sure this just inherits colors from the actual sidebar. Need to specify it explicitaly since browser have a default for `color` on button that would prevent the `inherit` otherwise.